sql/hive/src/test/scala/org/apache/spark/sql/hive/orc/OrcSourceSuite.scala

```diff
@@ -38,12 +39,12 @@ abstract class OrcSuite extends QueryTest with TestHiveSingleton with BeforeAndA
     super.beforeAll()
 
     orcTableAsDir = File.createTempFile("orctests", "sparksql")
-    orcTableAsDir.delete()
+    Utils.deleteRecursively(orcTableAsDir)
```

Comment:
Use `Utils.createTempDir` rather than this which is hacky. It will be automatically cleaned up too. We shouldn't have to reinvent this logic